cypress/e2e/dashboard.cy.ts

```diff
@@ -1,16 +1,25 @@
-import { Budget } from '../../src/types'
-import { createBudget, createEnvelope } from '../support/setup'
+import { first } from 'cypress/types/lodash'
+import { Budget, Envelope } from '../../src/types'
+import {
+  createBudget,
+  createAccount,
+  createEnvelope,
+  createTransaction,
+} from '../support/setup'
 
 describe('Dashboard', () => {
   beforeEach(() => {
     // prepare & select a budget
     cy.wrap(createBudget({ name: 'Dashboard Test' })).then((budget: Budget) => {
+      cy.wrap(budget).as('budget')
       cy.wrap(
         Cypress.Promise.all([
           createEnvelope({ name: 'First Envelope' }, budget),
           createEnvelope({ name: 'Second Envelope' }, budget),
         ])
-      ).then(() => {
+      ).then(([firstEnvelope, secondEnvelope]: Array<Envelope>) => {
```

Comment:
I prefer this syntax:
```suggestion
      ).then(([firstEnvelope, secondEnvelope]:Envelope[]) => {
```